Source/Shabby.csproj

```diff
@@ -1,63 +1,49 @@
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="0Harmony">
-      <HintPath>dlls\0Harmony.dll</HintPath>
-      <Private>false</Private>
-    </Reference>
-    <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>dlls\Assembly-CSharp.dll</HintPath>
-      <Private>false</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="UnityEngine.AssetBundleModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>dlls\UnityEngine.AssetBundleModule.dll</HintPath>
-      <Private>false</Private>
-    </Reference>
-    <Reference Include="UnityEngine.CoreModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>dlls\UnityEngine.CoreModule.dll</HintPath>
-      <Private>false</Private>
-    </Reference>
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Shabby.cs" />
-    <Compile Include="ShabLoader.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include=".editorconfig" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+﻿<?xml version="1.0" encoding="utf-8"?><Project Sdk="Microsoft.NET.Sdk">
+<PropertyGroup>
+  <TargetFramework>net48</TargetFramework>
+  <LangVersion>8</LangVersion>
+  <IsPackable>false</IsPackable>
+  <PlatformTarget>x64</PlatformTarget>
+  <NoWarn>1701;1702;CS0649;CS1591</NoWarn>
+  <AssemblyVersion>9.9.9</AssemblyVersion>
+</PropertyGroup>
+
+<ItemGroup>
+  <Reference Include="0Harmony, Culture=neutral, PublicKeyToken=null">
+    <HintPath>dlls/0Harmony.dll</HintPath>
+  </Reference>
+  <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
+    <HintPath>dlls/Assembly-CSharp.dll</HintPath>
+  </Reference>
+  <Reference Include="UnityEngine, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
+    <HintPath>dlls/UnityEngine.dll</HintPath>
+  </Reference>
+  <Reference Include="UnityEngine.AssetBundleModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
+    <HintPath>dlls/UnityEngine.AssetBundleModule.dll</HintPath>
+  </Reference>
+  <Reference Include="UnityEngine.CoreModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
+    <HintPath>dlls\UnityEngine.CoreModule.dll</HintPath>
+  </Reference>
+</ItemGroup>
+
+<ItemGroup>
+  <Compile Remove="dlls/**" />
+</ItemGroup>
+
+<ItemGroup>
+  <EmbeddedResource Remove="dlls/**" />
+</ItemGroup>
+
+<ItemGroup>
+  <None Remove="dlls/**" />
+</ItemGroup>
+
+<ItemGroup>
+  <None Include="..\.editorconfig" />
+</ItemGroup>
+<Target Name="PostBuild" AfterTargets="PostBuildEvent">
+  <Exec Command="/bin/cp -v '$(OutDir)Shabby.dll' '$(SolutionDir)../GameData/Shabby/Plugins/Shabby.dll'" IgnoreExitCode="true" />
+  <!--Fuck you MSBuild stop trying to run CMD.exe on macOS-->
```

Comment:
Just use the standard MSBuild tooling instead...
https://docs.microsoft.com/en-us/visualstudio/msbuild/copy-task?view=vs-2022